docs/configuration.md

```diff
@@ -72,14 +72,15 @@ If the API offers autoconfiguration data (e.g. through the [`x-cli-config` OpenA
 Once an API is configured, you can start using it by using its short name. For example, given an API named `example`:
 
 ```bash
-# If it has an API service description, call an operation
-$ restish example list-items
+# If it has an API service description, call an operation:
+$ restish example list-images
 
-# If there is no API description you can still use persistent headers and auth
-$ restish example/items
+# If there is no API description you can still use persistent headers, auth,
+# and the API short-name in URLs:
+$ restish example/images
 
 # It also works for full URIs, e.g. auth will be applied to:
-$ restish https://api.example.com/items
+$ restish https://api.example.com/images
```

Comment:
Was the `example.com` URL kept here deliberately? I'm asking because in [line 180 of guide.md](https://github.com/danielgtaylor/restish/pull/113/files#diff-07fdd026b11c494c3b62c97f764f6939803c453389ebefd6b05e6f187d44859aR180) you show how to register precisely an API named "example" whose base URL is `api.rest.sh`:

> Register a new API called `example`
> `$ restish api configure example https://api.rest.sh`

Reply:
Good catch! Fixed via https://github.com/danielgtaylor/restish/pull/114